docs/source/reference/config-sudo.md

```diff
@@ -163,7 +165,8 @@ instructions](http://ubuntuforums.org/showthread.php?t=992706).
 
 ### Shadow group (FreeBSD)
 
-**NOTE:** This has not been tested and may not work as expected.
+**NOTE:** This has not been tested on FreeBSD and may not work as expected on
```

Comment:
Add:

Do not use in production without verifying that it works properly.

Reply:
definitely worth making that as in-your-face as possible :)